### PR TITLE
chore: update to tooling v2

### LINKED ISSUE #3000: Update the Component Explorer to use FAST tooling v2
## Description

The Component Explorer is still pegged to FAST tooling v1. In order to take advantage of the new perf improvements and flexibility, it should be updated to FAST tooling v2.

## Requirements

- Update the dependencies to the latest versions of `@microsoft/fast-tooling` and `@microsoft/fast-tooling-react`
- Update the code by initializing the message system and all other related work to preserve functionality

### PR DESCRIPTION
**Pull request by janechu**
# Description

Updates the component explorer to FAST tooling v2.

This is a work in progress with various fixes needed before deployment can move out of staging.

Resolves #3000 

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/fast-component-explorer/webpack.config.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = (env, args) => {
         devtool: isProduction ? "none" : "inline-source-map",
         entry: {
             main: path.resolve(appDir, "index.tsx"),
-            serviceWorker: path.resolve(appDir, "service-worker-registration.ts"),
+            // serviceWorker: path.resolve(appDir, "service-worker-registration.ts"),
```
Comment (awentzel) [inline sites/fast-component-explorer/webpack.config.js:19]: This should either have a todo task linking to a github item or be removed all together.

File: sites/fast-component-explorer/webpack.config.js [hunk 2]
```diff
@@ -28,24 +28,23 @@ module.exports = (env, args) => {
...
-                                /[\\/]node_modules[\\/](.*?)([\\/]|$)/
-                            )[1];
-
-                            // npm package names are URL-safe, but some servers don't like @ symbols
-                            return `npm.${packageName.replace("@", "")}`;
-                        },
-                    },
-                },
-            },
+            // runtimeChunk: "single",
```
Comment (awentzel) [inline sites/fast-component-explorer/webpack.config.js:31]: What's the reason to comment out so much and committing these changes?

Comment (janechu) [reply sites/fast-component-explorer/webpack.config.js]: Updated, this webpack now mirrors the `fast-creator` webpack with comments to indicate why certain sections are commented out.

**Comment by janechu** [comment]
As a note, there are a lot of shared logic between `fast-creator` and `fast-component-explorer`, both of these sites are not intended for publishing until #3025 has been completed.

**Comment by janechu** [comment]
Closing this in order to re-open into a feature branch.

**Pull request closed without merging**